npc/re/jobs/novice/academy.txt

```diff
@@ -13,75 +13,241 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
+		navigateto("int_land", NAV_NONE, 1); // individual map name not required
+		izintspawn = 1;
+	}
+	end;
+}
+
+//== Wounded Swordsman (iz_int) =================================
+iz_int,56,32,3	script	Wounded Swordsman#izint	4_TOWER_01,8,8,{
+	if (!@display) {
+		classchange 4_TOWER_02, "", bc_self;
+		npctalk "Ohhhhhhh....", "", bc_self;
+		@display = 1;
+		end;
+	}
+	switch( isbegin_quest(21001) ) {
+	case 0:
+		mes "[Wounded]";
+		mes "Wow! Thanks a lot!";
+		mes "I don't know how this happened to our ship";
+		mes "but we should go to see the captain.";
+		next;
+		mes "[Wounded]";
+		mes "... ohh, it seems my body is injured too much.";
+		mes "Maybe you can go without me?";
+		setquest 21001;
+		next;
+		break;
+	case 1:
+		mes "[Wounded]";
+		mes "Luckily the water has not entered the ship.";
+		mes "I will wait here, you should go see the captain.";
+		next;
+		mes "[Wounded]";
+		mes "I'm fine! I'm fine!";
+		mes "Don't worry about me, you should really go to the captain!";
+		next;
+		break;
+	case 2:
+		break;
+	}
+	cutin "tutorial02",4;
+	mes "^4d4dff!- Information -!^000000";
+	mes "NPC Quest Received.";
```

Comment:
This line is only displayed when you get the quest. It will not be displayed if you talk to the NPC again.